DTM - Groups as assets (#2792)
SoftwareAG/c8y-docs: content/dtm/asset-hierarchy-bundle/group-assets.md

---
weight: 50

> title: Groups as Assets

Reviewer: ```suggestion
title: Asset Model for Groups
```

> title: Asset Model for Groups

Reviewer 2: ```suggestion
title: Asset model for groups
```

layout: redirect
---

> The default **Group** asset model allows any group to be treated as an asset, providing a unified, flexible, and consistent approach to managing assets.

Reviewer 2: ```suggestion
The default group asset model allows any group to be treated as an asset, providing a unified, flexible, and consistent approach to managing assets.
```

Reviewer 2: Bold formatting is only used for elements in the UI. Here it refers to a concept.

> To create an asset from the default group asset model in the **New asset** page, select the **Group (Default)** option from the **Choose asset model** dropdown menu on the top left. For information on how to create an asset, refer to [To create an asset](/dtm/asset-hierarchy/#to-create-an-asset). 

Reviewer 2: @lara-c8y One more thing. We do usually not use "choose", if we mean "select". But this would have to be changed in the UI. For now, the documentation is correct.

> To create an asset from the default **Group** asset model, in the **New asset** page, select the **Group (Default)** option from the **Choose asset model** dropdown menu on the top left. For information on how to create an asset, refer [To create an asset](/dtm/asset-hierarchy/#to-create-an-asset). 

Reviewer 2: ```suggestion
To create an asset from the default group asset model in the **New asset** page, select the **Group (Default)** option from the **Choose asset model** dropdown menu on the top left. For information on how to create an asset, refer to [To create an asset](/dtm/asset-hierarchy/#to-create-an-asset). 
```

##### Characteristics of a group asset

Since the default **Group** asset model does not enforce hierarchy, any group asset can be added to another group asset. However, only root assets created using models other than the default **Group** asset model are allowed under a group asset. 

> Once added to a group asset, a non-group asset can be moved back to the root level.

Reviewer: I couldnt understand what the above line meant.

Author: I meant to convey that if a group or an asset is added to another group, it can be moved outside of the group using the 'Move to root' option introduced in asset movement. Now that I think about it, the above sentence doesn't covey the information fully. Here is an improvement
```suggestion
Any asset added to a group asset can be moved out of it using the "Move to the root" option in the **Move assets** dialog.
```

> Any asset added to a group asset can be moved out of it using the "Move to the root" option in the **Move assets** dialog.

Reviewer 2: ```suggestion
Any asset added to a group asset can be moved out of it using the **Move to the root** option in the **Move assets** dialog.
```

{{< c8y-admon-info>}}

> - The Bulk import feature does not support creating group assets in bulk. 

Reviewer: ```suggestion
-  Currently, the [bulk import ](/dtm/asset-hierarchy/#creating-assets-bulk-import) feature does not support creating groups in bulk.
```

> -  Currently, the [bulk import ](/dtm/asset-hierarchy/#creating-assets-bulk-import) feature does not support creating groups in bulk.

Reviewer 2: ```suggestion
-  Currently, the [bulk import](/dtm/asset-hierarchy/#creating-assets-bulk-import) feature does not support creating groups in bulk.
```

> - Groups created in {{< product-c8y-iot >}} cockpit application are not listed in the Digital Twin Manager application.

Reviewer: not sure why this is not supported or this is only a problem until the customer create 1 asset after the introduction of updated microservice.

> - Groups created in {{< product-c8y-iot >}} cockpit application are not listed in the Digital Twin Manager application.

Reviewer 2: ```suggestion
- Groups created in the {{< product-c8y-iot >}} Cockpit application are not listed in the Digital Twin Manager application.
```
